dask_sql/physical/utils/groupby.py

```diff
@@ -20,7 +20,7 @@ def get_groupby_with_nulls_cols(
     group_columns_and_nulls = []
     for group_column in group_columns:
         # the ~ makes NaN come first
-        is_null_column = ~(group_column.isnull())
+        is_null_column = group_column.isnull()
```

Comment:
Noting that I removed the inverse operation here so that the slow path groupby agg is consistent with dask/dask-cudf's; this should also speed things up a little